aspnetcore/mvc/views/razor.md

```diff
@@ -442,43 +440,111 @@ public class _Views_Something_cshtml : RazorPage<dynamic>
 
 Later in this article, the section [Inspect the Razor C# class generated for a view](#inspect-the-razor-c-class-generated-for-a-view) explains how to view this generated class.
 
-<a name="using"></a>
+### \@attribute
 
-### @using
+The `@attribute` directive adds the given attribute to the class of the generated page or view. The following example adds the `[Authorize]` attribute:
 
-The `@using` directive adds the C# `using` directive to the generated view:
+```cshtml
+@attribute [Authorize]
+```
 
-[!code-cshtml[](razor/sample/Views/Home/Contact9.cshtml)]
+::: moniker range=">= aspnetcore-3.0"
 
-### @model
+### \@code
 
-The `@model` directive specifies the type of the model passed to a view:
+*This scenario only applies to Razor components (.razor).*
```

Comment:
Create a new H2 at the end of the file 

## Razor components specific

This scenario only applies to Razor components (.razor)

Move all 7 of your `This scenario only applies to Razor components (.razor)` to that section.

Reply:
Ping @danroth27 @NTaylorMullen for more info here on future plans. (You can skip to the last line if you want. :wink:)

I'm concerned about the layout: There are *directives* and *directive attributes*, so the topic might end up with four sections depending on what the framework offers down the road if we split out RC right now:

* RP/views
  * Directives
  * Directive attributes
* RC
  * Directives
  * Directive attributes

I was under the impression that at least some of the Razor Component-only scenarios (directive attributes) would be available for RP/views one day. Therefore, I structured the content to handle those framework enhancements should they occur ...

* Directives (alphabetical with qualifying lines for RP/view-only and RC-only scenarios)
* Directive attributes (alphabetical with qualifying lines for RP/view-only and RC-only scenarios)

The current layout provides for straightforward updates ...

* Just remove the 'This is RC only' line for any of the scenarios added on this PR when one or more become available for RP/views in the future.
* In the future, a scenario is added by the framework+docs to either the *Directives* or *Directive attributes* sections and ...
  * No qualifying line indicates that the scenario applies to both.
  * Use the 'This is RP/views only' line.
  * Use 'This is RC only' line.

Couldn't be simpler imo ... for the readers and the team.

If we go with a RC-specific section and some of the RC-only scenarios are extended to cover RP/views, duplicate content will result or the layout will need to go to what I have on this PR.

**TL;DR** ... *the 9-volume book!* :smile: 

**Will some or all of these new Razor Components Razor syntax scenarios (perhaps just the directive attributes) be coming to a Razor Pages page or MVC view _near you?!_** :point_left: If **_No! Never!_**, then let's try the RC-only section approach (but I'm also concerned about the 'end of the file' location ... that seems out-of-order). If **_Sure thing! Why the heck not?!_** :tada:, then I recommend the current approach on the PR now.